recipes/code-aster/meta.yaml

```diff
@@ -0,0 +1,111 @@
+{% set name = "code-aster" %}
+{% set version = "14.6" %}
+{% set minor_version = "11" %}
+{% set sha256 = "107ed98d10bc05866bdb3053f088fb0f51276ea14f7b615ec0d410a4b147ef12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - fn: {{ name }}_{{ version }}.{{ minor_version }}.tar.gz
+    url: https://gitlab.com/codeaster/src/-/archive/{{ version }}.{{ minor_version }}/src-{{ version }}.{{ minor_version }}.tar.gz
+    sha256: {{ sha256 }}
+  - url: https://downloads.sourceforge.net/project/prereq/metis-5.1.0_aster4.tar.gz
+    sha256: 1b295f9ce260c19d2d93fb2430be4139496c56f58f74ef7956ccd26c854fcc2e
+    folder: metis-aster
+  - url: http://graal.ens-lyon.fr/MUMPS/MUMPS_5.1.2.tar.gz
+    sha256: eb345cda145da9aea01b851d17e54e7eef08e16bfa148100ac1f7f046cd42ae9
+    folder: mumps-aster
+    patches:
+      - contrib/mumps-aster-diff.patch
+  - url: https://netcologne.dl.sourceforge.net/project/prereq/homard-11.12_aster2.tar.gz
+    sha256: 80425cee117349df8425f801f9a82aadb85fd41a95af25b4397f0e69dd655304
+    folder: homard
+    
+build:
+  number: 0
+  skip: true  # [py>=39 or osx or win or (python_impl == 'pypy')]
+  detect_binary_files_with_prefix: True
+  run_exports:
+    - gcc 8  # [linux]
+    - gxx 8  # [linux]
+    - gfortran 8  # [linux]
+  ignore_run_exports:
+    - numpy
```

Comment:
Why would numpy be needed at build time, but not also run time? If you're linking against the NumPy C API, then wouldn't you want numpy available at run time?

Reply:
I am not totally sure that it links against the numpy C API, it does check for it at build time, it might be used by the python code only ? In any case, I removed the ignore, I agree with you